components/uArchARM/CoreModel/registers.cpp

```diff
@@ -130,20 +127,18 @@ PhysicalMap &CoreImpl::mapTable(eRegisterType aMapTable) {
 
 mapped_reg CoreImpl::map(reg aReg) {
   FLEXUS_PROFILE();
-  reg reg = theWindowMap.rotate(aReg);
```

Comment:
This thing seems like a huge change to remove the call to `rotate()`. Was it truly doing nothing at all before this?

Reply:
I am pretty sure the rotate call was only required for the register window implementation.
Is it not the case?

Reply:
Yeah, I think it did not end up doing anything.

Reply:
Can you do me a favour please and just check it quickly before I merge it?

Reply:
Yes, I verified it.
There is a rotate function, but because all the register window parameters were set to zero, it did not end up doing anything.